lib/src/scss-shared/_nx-variables.scss

```diff
@@ -32,6 +32,8 @@ $nx-spacing-l: 24px;
 $nx-spacing-xl: 32px;
 $nx-spacing-xxl: 64px;
 
+$nx-flow-content-spacing: $nx-spacing-xl;
```

Comment:
This could use a comment to explain what it's for.  Especially since I thought we wanted to spacing between flow elements to be 24px, not 32.

Reply:
> This could use a comment to explain what it's for. Especially since I thought we wanted to spacing between flow elements to be 24px, not 32.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/0a9dcf5bf0601b60f58ac9a27b86dcc0bfdc8166